### DISCUSSION TITLE
`za` for toggle folds?

### DISCUSSION THREAD

**Discussion by sharpchen** [JetBrains/ideavim#1024] [Q&A]
Is it available or not implemented yet?

**Comment by citizenmatt** [comment]
I think it's part of 2.16.0, but I'd have to check properly

**Reply by sharpchen** [reply]
Toggle Folding action doesn't work too, might not be an ideavim issue

**Reply by citizenmatt** [reply]
See, that's why I should have checked properly 😬 I was looking at dates instead of commits, sorry. It's part of 2.17.0-eap.1. [The readme](https://github.com/JetBrains/ideavim?tab=readme-ov-file#get-early-access) explains how to use the EAP preview version.
